server/index.js

```diff
@@ -40,6 +38,109 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const sql = `
+    select *
+    from "carts"
+  `;
+  db.query(sql)
+    .then(result => {
+      if (!req.session.cartId) {
+        res.json([{}]);
+      } else {
+        const sql9000 = `
+        select "c"."cartItemId",
+                "c"."price",
+                "p"."productId",
+                "p"."image",
+                "p"."name",
+                "p"."shortDescription"
+        from "cartItems" as "c"
+        join "products" as "p" using ("productId")
+        where "c"."cartId" = ${req.session.cartId}
+        `;
```

Comment:
Ok, two things here. First, you don't need to `select * from "carts"` since we are only interested in `cartItems`. Also, if you stick values directly in your SQL like this.
```js
        where "c"."cartId" = ${req.session.cartId}
```
You app is vulnerable to SQL injection attacks. Please see the SQL injection video and `node-postgres` docs on paramaterized queries. These items are also covered in `sgt-back-end`.